Lombiq.HelpfulLibraries.LinqToDb/ITableAccessor.cs

```diff
@@ -14,4 +14,13 @@ public interface ITableAccessor
     /// <returns>Linq2db table for query building purposes.</returns>
     ITable<T> GetTable<T>()
         where T : class;
+
+    /// <summary>
+    /// Returns a table for query building purposes.
+    /// </summary>
+    /// <typeparam name="T">Type of the object representing the table in the database.</typeparam>
+    /// <param name="collectionName">Name of the collection.</param>
+    /// <returns>Linq2db table for query building purposes.</returns>
+    ITable<T> GetTable<T>(string collectionName)
```

Comment:
I'd just add the parameter to the existing method.

Also, add a mention of this (no need to demonstrate it since it's super simple) to `LinqToDbSamplesController` to aid discovery.

Reply:
Should be optional (null by default)?

Reply:
Should be optional (null by default)?

Reply:
Yes.